holoviews/element/raster.py

```diff
@@ -487,135 +401,37 @@ def closest(self, coords=[], **kwargs):
             return [getter(self.closest_cell_center(*el)) for el in coords]
 
 
-    def __getitem__(self, coords):
+    def range(self, dim, data_range=True):
+        idx = self.get_dimension_index(dim)
+        dimension = self.get_dimension(dim)
+        rng = super(Image, self).range(dim, data_range)
```

Comment:
I would deconstruct straight to ``low, high`` instead of introducing the weird ``rng`` name.